Anytype/Sources/PresentationLayer/Common/SwiftUI/IconView/Common/ImageCharIconView.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import SwiftUI
+
+struct ImageCharIconView: View {
+    
+    private struct Config {
+        let side: CGFloat
+        let fontSize: CGFloat
+
+        static let zero = Config(side: 0, fontSize: 0)
+    }
+
+    private static let configs = [
+        Config(side: 16, fontSize: 12),
+        Config(side: 18, fontSize: 14),
+        Config(side: 40, fontSize: 24),
+        Config(side: 48, fontSize: 30),
+        Config(side: 64, fontSize: 36),
+        Config(side: 80, fontSize: 48)
+    ].sorted(by: { $0.side > $1.side }) // Order by DESK side for simple search
```

Comment:
Order by DES**C** side for simple search?

Reply:
fixed